### DISCUSSION TITLE
blog/configurar-un-entorno-java

### DISCUSSION THREAD

**Discussion by giscus** [trevortylerlee/astro-micro#63] [Announcements]
# blog/configurar-un-entorno-java

Guía completa para configurar tu entorno de desarrollo Java. Desde la instalación del JDK hasta la creación de proyectos avanzados, aquí encontrarás todo lo que necesitas para comenzar a programar en Java.

http://localhost:4321/blog/configurar-un-entorno-java

**Comment by moyis** [comment]
Testing comments!

**Reply by trevortylerlee** [reply]
Yay!
